pennylane_lightning/core/src/simulators/lightning_qubit/catalyst/CMakeLists.txt

```diff
@@ -18,6 +18,9 @@ target_link_libraries(lightning_qubit_catalyst PUBLIC  lightning_compile_options
                                                        lightning_qubit_observables
 )
 
+set(CMAKE_BUILD_RPATH_USE_ORIGIN ON)
+set_target_properties(lightning_qubit_catalyst PROPERTIES BUILD_RPATH "$ORIGIN/../scipy_openblas32/lib:${SCIPY_OPENBLAS32_LIB_PATH}")
```

Comment:
Just double-checking here: shall we check the OS before setting the target properties?

Reply:
yea, tested it for `macos` and `linux`. Both work. Havn't got a chance to test it against `win32`